android/keystore_1.md

```diff
@@ -0,0 +1,54 @@
+# Android Keystore
+
```

Comment:
Maybe a one liner giving the motivation for this chapter ? Something in the lines, 'Your app is ready and you want to get it to your clients. In order to  do so, you need to sign your app. For that, you need to pass through the keystores'